docs/README.cursor.md

```
@@ -0,0 +1,220 @@
+# Superpowers for Cursor
+
+Complete guide for using Superpowers with Cursor IDE.
+
+## Quick Install
+
+Tell Cursor:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)